jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -118,7 +138,7 @@ pipeline {
 
                     if(OPENSEARCH_ARTIFACT_URL) {
                         ARCHITECTURE = url_paths.values().find { it.contains("x64") } ? "x64" : "arm64"
-                        DISTRIBUTION = url_paths.values().find { it.contains("tar.gz") } ? "tar" : (url_paths.values().find { it.contains("repo") } ? "yum" : "rpm")
+                        DISTRIBUTION = url_paths.values().find { it.contains("tar.gz") } ? "tar" : url_paths.values().find { it.contains("zip") } ? "zip" : url_paths.values().find { it.contains("rpm") } ? "rpm" : "deb"
```

Comment:
There should be another key-value pair for this such as the dist:image pair instead of writing such a long line.

Reply:
Seems like it is waiting for this:
* 
https://github.com/opensearch-project/opensearch-build-libraries/pull/388